packages/engine/src/datastore/table/proxy.rs

```diff
@@ -37,15 +38,16 @@ use crate::datastore::{
     table::{pool::BatchPool, state::view::StatePools},
 };
 
-/// A thread-sendable guard for reading a batch, see module-level documentation for more reasoning.
+/// A thread-[`Send`]able guard for reading a batch, see module-level documentation for more
```

Comment:
Does this doc-link work? Because the top level one doesn't have square brackets

Reply:
Yup, checked them :)